SDK/AppCenterAnalytics/Microsoft.AppCenter.Analytics.Windows.Shared/Channel/SessionTracker.cs

```diff
@@ -86,6 +86,14 @@ public void Resume()
             }
         }
 
+        public void Stop()
+        {
+            lock (_lockObject)
+            {
+                UpdateSessionId(Guid.Empty);
```

Comment:
Test missing for that one.
Also can we get rid of LastSid now that we have this mechanism? Several tests depend on it so not easy to refactor.